src/more-infos/more-info-camera.html

```diff
@@ -29,21 +29,26 @@
     stateObj: {
       type: Object,
     },
+
+    isVisible: {
+      type: Boolean,
+      value: true,
+    },
   },
 
   imageLoaded: function () {
     this.fire('iron-resize');
   },
 
-  computeCameraImageUrl: function (hass, stateObj) {
+  computeCameraImageUrl: function (hass, stateObj, isVisible) {
     if (hass.demo) {
       return '/demo/webcam.jpg';
-    } else if (stateObj) {
+    } else if (stateObj && isVisible) {
```

Comment:
I'm wondering now, I thought that this logic of `isVisible` was provided by the fact that stateObj is set to null when the more info dialog is hidden, is that no longer the case?